content/articles/2019-04-22-dplyr-0-8-1.Rmarkdown

```diff
@@ -0,0 +1,105 @@
ub.com/grabear), [&#x0040;grandtiger](https://github.com/grandtiger), [&#x0040;gregorp](https://github.com/gregorp), [&#x0040;hadley](https://github.com/hadley), [&#x0040;hanyroze](https://github.com/hanyroze), [&#x0040;hidekoji](https://github.com/hidekoji), [&#x0040;huftis](https://github.com/huftis), [&#x0040;iago-pssjd](https://github.com/iago-pssjd), [&#x0040;javierluraschi](https://github.com/javierluraschi), [&#x0040;jennybc](https://github.com/jennybc), [&#x0040;jgellar](https://github.com/jgellar), [&#x0040;jhrcook](https://github.com/jhrcook), [&#x0040;jimhester](https://github.com/jimhester), [&#x0040;joel23888](https://github.com/joel23888), [&#x0040;JohnMount](https://github.com/JohnMount), [&#x0040;johnmous](https://github.com/johnmous), [&#x0040;jonathan-g](https://github.com/jonathan-g), [&#x0040;jwbeck97](https://github.com/jwbeck97), [&#x0040;jzadra](https://github.com/jzadra), [&#x0040;karimn](https://github.com/karimn), [&#x0040;kendonB](https://github.com/kendonB), [&#x0040;koncina](https://github.com/koncina), [&#x0040;kperkins](https://github.com/kperkins), [&#x0040;kputschko](https://github.com/kputschko), [&#x0040;krlmlr](https://github.com/krlmlr), [&#x0040;kyzphong](https://github.com/kyzphong), [&#x0040;lionel-](https://github.com/lionel-), [&#x0040;llrs](https://github.com/llrs), [&#x0040;mariodejung](https://github.com/mariodejung), [&#x0040;MichaelAdolph](https://github.com/MichaelAdolph), [&#x0040;michaelwhammer](https://github.com/michaelwhammer), [&#x0040;MilesMcBain](https://github.com/MilesMcBain), [&#x0040;mjherold](https://github.com/mjherold), [&#x0040;moodymudskipper](https://github.com/moodymudskipper), [&#x0040;msberends](https://github.com/msberends), [&#x0040;mvkorpel](https://github.com/mvkorpel), [&#x0040;nathancday](https://github.com/nathancday), [&#x0040;nicokuz](https://github.com/nicokuz), [&#x0040;nolistic](https://github.com/nolistic), [&#x0040;oscci](https://github.com/oscci), [&#x0040;paulponcet](https://github.com/paulponcet), [&#x0040;PhilippRuchser](https://github.com/PhilippRuchser), [&#x0040;philstraforelli](https://github.com/philstraforelli), [&#x0040;psychometrician](https://github.com/psychometrician), [&#x0040;Ranonymous](https://github.com/Ranonymous), [&#x0040;rinebob](https://github.com/rinebob), [&#x0040;romagnolid](https://github.com/romagnolid), [&#x0040;romainfrancois](https://github.com/romainfrancois), [&#x0040;rvg02010](https://github.com/rvg02010), [&#x0040;slyrus](https://github.com/slyrus), [&#x0040;snp](https://github.com/snp), [&#x0040;sowla](https://github.com/sowla), [&#x0040;ThiAmm](https://github.com/ThiAmm), [&#x0040;thothal](https://github.com/thothal), [&#x0040;wfmackey](https://github.com/wfmackey), [&#x0040;will458](https://github.com/will458), [&#x0040;wkdavis](https://github.com/wkdavis), [&#x0040;yutannihilation](https://github.com/yutannihilation), [&#x0040;ZahraEconomist](https://github.com/ZahraEconomist), and [&#x0040;zooman](https://github.com/zooman)
```

Comment:
```suggestion
[&#x0040;abalter](https://github.com/abalter), [&#x0040;ambevill](https://github.com/ambevill), [&#x0040;amitusa17](https://github.com/amitusa17), [&#x0040;AntoineHffmnn](https://github.com/AntoineHffmnn), [&#x0040;anuj2054](https://github.com/anuj2054), [&#x0040;batpigandme](https://github.com/batpigandme), [&#x0040;behrman](https://github.com/behrman), [&#x0040;billdenney](https://github.com/billdenney), [&#x0040;burchill](https://github.com/burchill), [&#x0040;cgrandin](https://github.com/cgrandin), [&#x0040;clemenshug](https://github.com/clemenshug), [&#x0040;codetrainee](https://github.com/codetrainee), [&#x0040;ColinFay](https://github.com/ColinFay), [&#x0040;dan-reznik](https://github.com/dan-reznik), [&#x0040;davidsjoberg](https://github.com/davidsjoberg), [&#x0040;DesiQuintans](https://github.com/DesiQuintans), [&#x0040;dirkschumacher](https://github.com/dirkschumacher), [&#x0040;earowang](https://github.com/earowang), [&#x0040;echasnovski](https://github.com/echasnovski), [&#x0040;eipi10](https://github.com/eipi10), [&#x0040;grabear](https://github.com/grabear), [&#x0040;grandtiger](https://github.com/grandtiger), [&#x0040;gregorp](https://github.com/gregorp), [&#x0040;hadley](https://github.com/hadley), [&#x0040;hanyroze](https://github.com/hanyroze), [&#x0040;hidekoji](https://github.com/hidekoji), [&#x0040;huftis](https://github.com/huftis), [&#x0040;iago-pssjd](https://github.com/iago-pssjd), [&#x0040;javierluraschi](https://github.com/javierluraschi), [&#x0040;jennybc](https://github.com/jennybc), [&#x0040;jgellar](https://github.com/jgellar), [&#x0040;jhrcook](https://github.com/jhrcook), [&#x0040;jimhester](https://github.com/jimhester), [&#x0040;joel23888](https://github.com/joel23888), [&#x0040;JohnMount](https://github.com/JohnMount), [&#x0040;johnmous](https://github.com/johnmous), [&#x0040;jonathan-g](https://github.com/jonathan-g), [&#x0040;jwbeck97](https://github.com/jwbeck97), [&#x0040;jzadra](https://github.com/jzadra), [&#x0040;karimn](https://github.com/karimn), [&#x0040;kendonB](https://github.com/kendonB), [&#x0040;koncina](https://github.com/koncina), [&#x0040;kperkins](https://github.com/kperkins), [&#x0040;kputschko](https://github.com/kputschko), [&#x0040;krlmlr](https://github.com/krlmlr), [&#x0040;kyzphong](https://github.com/kyzphong), [&#x0040;lionel-](https://github.com/lionel-), [&#x0040;llrs](https://github.com/llrs), [&#x0040;mariodejung](https://github.com/mariodejung), [&#x0040;MichaelAdolph](https://github.com/MichaelAdolph), [&#x0040;michaelwhammer](https://github.com/michaelwhammer), [&#x0040;MilesMcBain](https://github.com/MilesMcBain), [&#x0040;mjherold](https://github.com/mjherold), [&#x0040;moodymudskipper](https://github.com/moodymudskipper), [&#x0040;msberends](https://github.com/msberends), [&#x0040;mvkorpel](https://github.com/mvkorpel), [&#x0040;nathancday](https://github.com/nathancday), [&#x0040;nicokuz](https://github.com/nicokuz), [&#x0040;nolistic](https://github.com/nolistic), [&#x0040;oscci](https://github.com/oscci), [&#x0040;paulponcet](https://github.com/paulponcet), [&#x0040;PhilippRuchser](https://github.com/PhilippRuchser), [&#x0040;philstraforelli](https://github.com/philstraforelli), [&#x0040;psychometrician](https://github.com/psychometrician), [&#x0040;Ranonymous](https://github.com/Ranonymous), [&#x0040;rinebob](https://github.com/rinebob), [&#x0040;romagnolid](https://github.com/romagnolid), [&#x0040;romainfrancois](https://github.com/romainfrancois), [&#x0040;rvg02010](https://github.com/rvg02010), [&#x0040;slyrus](https://github.com/slyrus), [&#x0040;snp](https://github.com/snp), [&#x0040;sowla](https://github.com/sowla), [&#x0040;ThiAmm](https://github.com/ThiAmm), [&#x0040;thothal](https://github.com/thothal), [&#x0040;wfmackey](https://github.com/wfmackey), [&#x0040;will458](https://github.com/will458), [&#x0040;wkdavis](https://github.com/wkdavis), [&#x0040;yutannihilation](https://github.com/yutannihilation), [&#x0040;ZahraEconomist](https://github.com/ZahraEconomist), and [&#x0040;zooman](https://github.com/zooman).
```

Reply:
Adds a period to the end of the list of people to thank. (Really, me trying out the PR-within-PR thing).